core/src/test/java/ai/timefold/solver/core/impl/heuristic/HeuristicConfigPolicyTestUtils.java

```diff
@@ -15,9 +15,8 @@ public static HeuristicConfigPolicy<TestdataSolution> buildHeuristicConfigPolicy
 
     public static <Solution_> HeuristicConfigPolicy<Solution_>
             buildHeuristicConfigPolicy(SolutionDescriptor<Solution_> solutionDescriptor) {
-        return new HeuristicConfigPolicy.Builder<>(EnvironmentMode.REPRODUCIBLE, null, null, null, null, new Random(), null,
-                solutionDescriptor, ClassInstanceCache.create())
-                .build();
+        return new HeuristicConfigPolicy.Builder<>(null, EnvironmentMode.REPRODUCIBLE, null, null, null, null, new Random(),
+                null, solutionDescriptor, ClassInstanceCache.create()).build();
```

Comment:
Isn't the point of a builder to not have these crazy constructors?

Reply:
Yes, it should just provide a fluid API. Let me fix that.